javascript/javascript1/week1/Housey-pricey.js

```diff
@@ -0,0 +1,33 @@
+//calculate house price
+let peter = {
+    depth:10,
+    width:8,
+    height:10,
+    areaOfGarden:100
+
+};
+let julia = {
+    depth:11,
+    width:5,
+    height:8,
+    areaOfGarden:70
+    
+};
+function calHousePrice(houseInfo){
+
+    let housePrice = (houseInfo.width * houseInfo.height * houseInfo.depth) * 2.5 * 1000 + (houseInfo.areaOfGarden * 300);
+    return housePrice;
```

Comment:
Because the function is properly named (`calHousePrice`), I would consider the `let housePrice` unnecessary.

Consider if you would find the following equaly readable:
```js
function calHousePrice(houseInfo){
    return (houseInfo.width * houseInfo.height * houseInfo.depth) * 2.5 * 1000 + (houseInfo.areaOfGarden * 300);
}
```
Because it is more important that it is readable to you than to me :wink: